libs/hmpb/layout/src/document_svg.tsx

```diff
@@ -48,6 +50,50 @@ export function SvgBubble({
   return <SvgRectangle {...props} />;
 }
 
+function splitIntoHtmlSpans(html: string): React.ReactNode[] {
+  // Remove any <html> and </html> tags in the line
+  // eslint-disable-next-line no-param-reassign
+  html = html.replace(/<\/?html>/, '');
+
+  // DOMParser breaks if there's a closing tag with no opening tag, so fix that
+  // It doesn't mind unclosed tags, so we don't need to fix those
```

Comment:
Why would we encounter closing tags with no opening tags. Is this to do with splitting into lines?

Reply:
Yep, lines get split higher up the stack

Reply:
I think it'd be easier for me to understand the intent of this code if there were some tests.